src/knex-tables.d.ts

```diff
@@ -183,12 +183,13 @@ declare module "knex/types/tables" {
     middle_name?: string;
     last_name: string;
     status: RemovalStatus;
+    manually_resolved: boolean;
     created_at: Date;
     updated_at: Date;
   }
   type OnerepScanResultOptionalColumns = Extract<
     keyof OnerepScanResultRow,
-    "middle_name"
+    "manually_resolved" | "middle_name"
```

Comment:
Should this be optional? We could make this a required column (with a default of `false`) if that makes more sense. Is the thinking more that the calling code won't always care about this value?